specs/interop/derivation.md

```diff
@@ -294,9 +297,10 @@ The expiry window is the time period after which an initiating message is no lon
 ### Depositing an Executing Message
 
 Deposit transactions (force inclusion transactions) give censorship resistance to layer two networks.
-It is possible to deposit an invalid executing message, forcing the sequencer to reorg. It would
+If it were possible to deposit an invalid executing message, this would force the sequencer to reorg. It would
 be fairly cheap to continuously deposit invalid executing messages through L1 and cause L2 liveness
-instability. A future upgrade will enable deposits to trigger executing messages.
+instability, therefore deposits are prevented from triggering executing messages.
+A future upgrade will enable deposits to trigger executing messages.
```

Comment:
```suggestion
```
I would omit this for now. We could maybe add a note that it "may" do that for future-compatibility concerns of the reader designing things against the spec, but we should not promise any changes.

Reply:
Replaced "will" with "may"